newsfragments/+py39.changed

```diff
@@ -0,0 +1 @@
+Support for Python 3.9 was deprecated and will be removed in our next planned release.
```

Comment:
why + format instead of using this pr number?

Reply:
yeah, it's a little weird. I debated suggesting we use `pull` instead of `issues` in the towncrier pr, but figured it didn't really matter either way.